demo-smart/cc-pricing-page/index.html

```diff
@@ -126,6 +126,13 @@ <h2>Redis</h2>
             </cc-smart-container>
         </div>
 
+        <div class="product">
+          <cc-smart-container context='{"productId":"mongodb-addon"}'>
+            <h2>MongoDB</h2>
+            <cc-pricing-product mode="addon" action="add"></cc-pricing-product>
+          </cc-smart-container>
+      </div>
```

Comment:
nitpick(non-blocking): the indentation feels off here :thinking:

Reply:
good catch :wink: